tools/run_satwnds/satwnds_new_subset_template.yaml

```diff
@@ -72,6 +72,11 @@ observations:
         - name: "ncols_conf"
           size: variables/percent_confidence.ncols
 
+      globals:
+        - name: "bufr_version@MetaData"
+          type: string
+          value: "310077"    
```

Comment:
It might make sense to make the name something like seq_version and and the value equivalent to the mnemonic for that sequence (GEOSTWND or whatever), up to you :)

Reply:
bufr_version, seq_version, or even bufr_seq_version - I'm fine with either choice.

I used "310077"  b/c it comes straight from WMO documents. In fact, it should be "3-10-077". GEOSTWND is more familiar to NCEP folks, I agree, but I don't think MetOffice/NRL /etc are familiar with our naming in Table D.

Let's get @emilyhcliu to chime in, she is leading the Obs Re-engineering effort and may have an opinion on this too.